tests/test_pyglmnet.py

```diff
@@ -345,3 +345,40 @@ def test_cdfast():
 def test_fetch_datasets():
     """Test fetching datasets."""
     datasets.fetch_community_crime_data('/tmp/glm-tools')
+
+
+def test_random_state_consistency():
+    """ Test model's random_state """
+
+    # Generate the dataset
+    n_samples, n_features = 1000, 10
+
+    beta0 = 1. / (np.float(n_features) + 1.) * np.random.normal(0.0, 1.0)
+    beta = 1. / (np.float(n_features) + 1.) * \
+        np.random.normal(0.0, 1.0, (n_features,))
+    Xtrain = np.random.normal(0.0, 1.0, [n_samples, n_features])
+
+    ytrain = simulate_glm("gaussian", beta0, beta, Xtrain,
+                          sample=False, random_state=42)
+
+    # Test simple glm
+    glm_a = GLM(distr="gaussian", random_state=1)
+    result_a = glm_a.fit_predict(Xtrain, ytrain)
+    glm_b = GLM(distr="gaussian", random_state=1)
+    result_b = glm_b.fit_predict(Xtrain, ytrain)
+    result_c = glm_b.fit_predict(Xtrain, ytrain)
+
+    # Consistency between two different models
+    assert_array_equal(result_a, result_b)
+    # Consistency between different run of the same model
+    assert_array_equal(result_b, result_c)
+
+    # Test also cross-validation
+    glm_cv_a = GLMCV(distr="gaussian", cv=3, random_state=1)
+    result_a_cv = glm_cv_a.fit_predict(Xtrain, ytrain)
+    glm_cv_b = GLMCV(distr="gaussian", cv=3, random_state=1)
+    result_b_cv = glm_cv_b.fit_predict(Xtrain, ytrain)
+    result_c_cv = glm_cv_b.fit_predict(Xtrain, ytrain)
```

Comment:
you can also call this `ypred_b` if you put the first assert statement above this line